clang/test/Driver/sycl-profile-update.cpp

```diff
@@ -0,0 +1,3 @@
+// Ensure that the profile update mode is set to 'atomic' when compiling SYCL code.
+// RUN: %clang -### -fsycl -fprofile-instr-generate -fcoverage-mapping %s 2>&1 | FileCheck %s
```

Comment:
Is this supported in `clang_cl` mode as well? If yes, please add a test case for the same.